cpp/biconnected_components_module/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+set(biconnected_components_module_src 
```

Comment:
Change the name of algorithm. `"CALL cpp_biconnected_components_module.biconnected_components()"` is way too long.